telegrambot/99-telegrambot.js

```diff
@@ -249,6 +249,12 @@ module.exports = function (RED) {
                                     msg.payload.path = path;
                                     node.send([msg, null]);
                                 });
+                            } else if (config.getFilePath && messageDetails.blob) {
+                                node.telegramBot.getFileLink(messageDetails.content).then(function(path) {
+                                    msg.payload.path = path;
+                                    node.send([msg, null]);
+                                });
+                            }
```

Comment:
I believe this brace is a junk, no?

Reply:
Well, the question is if the message should always contain the FileLink. Why should it be configurable?

Reply:
similar to the download.. getting the filelink would require another http call to the telegram bot.. so if it is not required, better not call it to speed up the request, no?

Reply:
@dvv what do you mean junk?